src/RestWrite.js

```diff
@@ -353,14 +353,16 @@ RestWrite.prototype.transformUser = function() {
   if (this.query) {
```

Comment:
I think this code simply relies on the fact that then() resolves to SUCCESS if the handler returns **undefined**. It would be cleaner to return a properly resolved promise.